core-bundle/src/Resources/contao/models/UserModel.php

```diff
@@ -45,18 +45,18 @@
  * @property string|array|null $formp
  * @property string|array|null $amg
  * @property string|boolean    $disable
- * @property string|integer    $start
- * @property string|integer    $stop
+ * @property integer           $start
+ * @property integer           $stop
```

Comment:
These are `varchar` in the database and thus they would still be returned as a string from there - and not converted to integer automatically, will they?

Reply:
On a side note: why _do_ we save those timestamps as a string in the first place? I guess it's just an historic remnant from Contao 2 and we could may be change the field type to `int(10) default NULL` and then alter the queries to

```diff
-start='' OR start<='$time'
+start IS NULL OR start<=$time
```

❓

Reply:
Yes, definitely a separate PR as this is a BC breaking change.